src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -262,13 +263,11 @@ def load_pytorch_state_dict_in_tf2_model(
 
     if _prefix is None:
         _prefix = ""
-    if tf_inputs is not None:
+    if tf_inputs:
```

Comment:
No we don't trust Python bool conversion magic and never will, so let's revert that change or add an or for the False test if it;s what you want to do here.

Reply:
lol that is dedication to duty, go lie down and talk to the hallucinations for a bit, I promise not to break the codebase while you're gone!